stdlib/experimental/influxdb/influxdb.flux

```diff
@@ -0,0 +1,18 @@
+package influxdb
+
+// api submits an HTTP request to the specified API path.
+// Returns HTTP status code, response headers, and body as a byte array.
+builtin api : (
```

Comment:
How is this function different from the `http.post` method? Could it just be a wrapper around that function instead of adding a new builtin? If not why not? Perhaps we could update `http.post` to enable that kind of workflow.

Reply:
The main difference is that this function is meant to support any request method, not just an HTTP POST. It can be used to create a resource via POST, retrieve that resource with a GET, remove it with a DELETE, etc.

Reply:
Makes sense. This got me thinking and I designed out an `http.do` function that would be a generic http request function. I think an API like this would work:

```
package http

builtin do : (
    url: string,
    method: string,
    ?headers: [string:string],
    ?timeout: duration,
) => {
    statusCode: int,
    body: bytes,
    headers: [string:string],
}

post = (
    url,
    headers: [:],
    timeout: 0s,
) => do(url:url, method: "POST", headers: headers, timeout: timeout)

get = (
    url,
    headers: [:],
    timeout: 0s,
) => do(url:url, method: "GET", headers: headers, timeout: timeout)
```